### DISCUSSION TITLE
Flet Designer: Drag and Drop

### DISCUSSION THREAD

**Discussion by ArcadeDesigns** [flet-dev/flet#1977] [Ideas]
Is there an on going project to develop a Flet designer like the PyQt5 Designer, with drag and drop components to speed up the creation of Flet application for web, mobile and desktop development, I think that will be cool.

**Comment by ndonkoHenri** [comment]
How do you find this https://github.com/SKbarbon/Flet_StoryBoard ?

**Comment by SKbarbon** [comment]
You're @ArcadeDesigns  welcome to try [Flet StoryBoard](https://github.com/SKbarbon/Flet_StoryBoard), There is a very huge upcoming main updates just on the way.
